mobile/src/modules/transactions/components/send/SendBurstForm.tsx

```diff
@@ -2,9 +2,11 @@ import { Account, SuggestedFees } from '@burstjs/core';
 import {
   convertAddressToNumericId,
   convertNQTStringToNumber,
+  convertNumberToNQTString,
   convertNumericIdToAddress,
   isBurstAddress,
-  isValid
+  isValid,
+  sumNQTStringToNumber
```

Comment:
I need to review/change its internal implementation as I fear numerical inaccuracies...you don't have to worry though :grin: